srcpkgs/swirc/template

```diff
@@ -1,19 +1,26 @@
 # Template file for 'swirc'
 pkgname=swirc
-version=3.3.0
+version=3.3.1
 revision=1
 build_style=configure
+configure_args="--without-libintl"
 make_install_args="PREFIX=/usr"
+make_install_target="install-no-lc-msgs"
```

Comment:
Why did you prefer to remove `libintl`?

Reply:
Is it okey to install stuff under /usr/local? Else main.cpp needs modification in the bindtextdomain() call. Maybe I can write a patch?

Reply:
For context: https://github.com/uhlin/swirc/blob/332223632e8667a1e649a491f20b9b0627928797/src/main.cpp#L507

Please try to write a patch, yes. The patch in our side can simply hardcode the path we install to, but if you'd like to upstream a change so it uses the path passed to Make, that'd be ideal. Or open an issue and request it :)